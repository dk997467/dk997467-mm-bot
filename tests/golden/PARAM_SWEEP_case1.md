PARAM SWEEP REPORT

| max_delta_ratio | impact_cap_ratio | min_interval_ms | tail_age_ms | net_bps | order_age_p95_ms | fill_rate | replace_rate_per_min | SAFE |
|-----------------|------------------|------------------|-------------|---------|-------------------|-----------|----------------------|------|
| 0.100000 | 0.060000 | 100.000000 | 600.000000 | 2.732400 | 296.100000 | 0.735000 | 180.000000 | SAFE |
| 0.100000 | 0.080000 | 100.000000 | 600.000000 | 2.766000 | 301.250000 | 0.735000 | 180.000000 | SAFE |
| 0.100000 | 0.100000 | 100.000000 | 600.000000 | 2.799600 | 306.400000 | 0.735000 | 180.000000 | SAFE |


