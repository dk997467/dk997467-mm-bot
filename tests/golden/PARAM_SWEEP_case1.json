{"grid":{"impact_cap_ratio":[0.06,0.08,0.1],"max_delta_ratio":[0.1,0.12,0.15],"min_interval_ms":[60,80,100],"tail_age_ms":[600,700,800]},"results":[{"metrics":{"fill_rate":0.735,"net_bps":2.7324,"order_age_p95_ms":296.1,"replace_rate_per_min":180.0},"params":{"impact_cap_ratio":0.06,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0}},{"metrics":{"fill_rate":0.735,"net_bps":2.766,"order_age_p95_ms":301.25,"replace_rate_per_min":180.0},"params":{"impact_cap_ratio":0.08,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0}},{"metrics":{"fill_rate":0.735,"net_bps":2.7996,"order_age_p95_ms":306.4,"replace_rate_per_min":180.0},"params":{"impact_cap_ratio":0.1,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0}}],"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"},"top3_by_net_bps_safe":[{"metrics":{"fill_rate":0.735,"net_bps":2.7324,"order_age_p95_ms":296.1,"replace_rate_per_min":180.0},"params":{"impact_cap_ratio":0.06,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0}},{"metrics":{"fill_rate":0.735,"net_bps":2.766,"order_age_p95_ms":301.25,"replace_rate_per_min":180.0},"params":{"impact_cap_ratio":0.08,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0}},{"metrics":{"fill_rate":0.735,"net_bps":2.7996,"order_age_p95_ms":306.4,"replace_rate_per_min":180.0},"params":{"impact_cap_ratio":0.1,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0}}]
}

